assets/js/modules/search-console/index.js

```diff
@@ -25,7 +25,7 @@ import domReady from '@wordpress/dom-ready';
  * Internal dependencies
  */
 import Modules from 'googlesitekit-modules';
-import Widgets from 'googlesitekit-widgets';
+import { registerWidget, WIDGET_WIDTHS } from 'googlesitekit-widgets';
```

Comment:
Nit-pick, but we always include the full objects for our "external" imports, let's keep that.

Reply:
Reverted.